tensorflow/python/kernel_tests/BUILD

```diff
@@ -3506,6 +3506,20 @@ cuda_py_test(
     xla_enable_strict_auto_jit = True,
 )
 
+cuda_py_test(
+    name = "normalize_op_test",
+    size = "medium",
+    srcs = ["normalize_op_test.py"],
+    additional_deps = [
+        "//third_party/py/numpy",
+        "//tensorflow/python:client_testlib",
+        "//tensorflow/python:framework_for_generated_wrappers",
+        "//tensorflow/python/ops:nn_impl",
```

Comment:
This should be //tensorflow/python/ops:nn

Reply:
Thanks ;)
That would have been my second attempt also.

Reply:
Why doesn't it find nn_impl?